<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TVForm</class>
 <widget class="QDialog" name="TVForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TVForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Producent:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ocena:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wysokosc ekranu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Szerokosc ekranu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Typ ekranu: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jasnosc ekranu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sygnal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Glosnosc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zasilanie :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Glosniki :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_producer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>domyslny</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_model">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>domyslny</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_screen">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>domyslny</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_rating">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_height">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_width">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_brightness">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_volume">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>522</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_power">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_pwrOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wlaczone</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_pwrOff">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wylaczone</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_signal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_signalOf">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wylaczony</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_signalOn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wlaczony</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_speakers">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_SpeakersOn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Podpiete</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_SpeakersOff">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Odpiete</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>TVForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_pwrOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton_signalOn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_SpeakersOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_volume</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_pwrOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_brightness</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_pwrOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton_signalOf</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_model</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TVForm</receiver>
   <slot>onTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_producer</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TVForm</receiver>
   <slot>onTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_screen</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TVForm</receiver>
   <slot>onTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_add</sender>
   <signal>clicked()</signal>
   <receiver>TVForm</receiver>
   <slot>submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChange()</slot>
  <slot>submit()</slot>
 </slots>
</ui>
